### DISCUSSION TITLE
search box at brew.sh finds nothing

### DISCUSSION THREAD

**Discussion by duanemoody** [Homebrew/discussions#4359] [Everyday usage]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

The homepage for brew.sh has a search box but it finds "No results for" anything entered, this has been checked on desktop and mobile in multiple browsers. The page's JS console shows no errors, just messages for each keypress. Also, ESC only clears the search box, it doesn't close the modal.

**Comment by FreQRiDeR** [comment]
Same, search box is broken across the board, it seems.

**Comment by p-linnane** [comment]
We are having issues with our Algolia crawler. We have an open case with them on this and hope it will be resolved soon.

cc @SMillerDev & @Bo98
